src/components/CodePreview/index.css

```diff
@@ -53,6 +53,10 @@
   background: var(--ic-architectural-white);
 }
 
+.ic-theme-dark .comp-zone {
```

Comment:
<img width="1413" alt="Screenshot of Custom Theme page" src="https://github.com/user-attachments/assets/3cd98549-a3b8-4bac-80a7-e680d28cc445">
The code preview doesn't respond to a change from dark mode to light mode.

Also why was #333 chosen for the background colour? In light mode, it is white so I'd expect the code preview to be our chosen dark mode colour: #17191C

Reply:
This applies to other instances of #333333